### PR TITLE
Targeted style to section-container

### LINKED ISSUE #546: Research Literature Page spacing issue
I noticed that there is only spacing on the last tab's content:

First tab:
image

Last tab:
image

@sdmcraft

### PR DESCRIPTION
**Pull request by sdmcraft**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after), along with a short summary of changes:

## Issue

Fixes #546 

## Changelog:
Targeted style to section-container

## Test URLs:
- Original: https://www.sunstar.com/research/literature
- Before: https://main--sunstar--hlxsites.hlx.live/research/literature
- After: https://i-546--sunstar--hlxsites.hlx.live/research/literature

## Library

- [ ] New Blocks introduced in this PR
      If yes, please provide details below
**Block Name** : For e.g. _cards_
- [ ] Documented in sidekick library

- [ ] New variations introduced in this PR
**Variation Name** :  For e.g. _cards (grid)_
- [ ] Documented in sidekick library

- [ ] New mixins introduced in this PR
**Mixin Name** :  For e.g. _compact, white_
- [ ] Documented in sidekick library

### CODE REVIEW

**Review by dnbute — APPROVED** [review]
lgtm

File: styles/styles.css [hunk 1]
```diff
@@ -1166,6 +1164,11 @@ main .section.with-background-image.align-top > .section-container, main .sectio
     margin-bottom: 1rem;
   }
 
+  /* section - static */
+  main .section.static > .section-container {
```
Comment (jindaliiita) [inline styles/styles.css:1168]: are we using static only at one place ?

**Comment by jindaliiita** [comment]
Overall LGTM apart from one comment
https://github.com/hlxsites/sunstar/pull/547/files#r1433883893

**Review by jindaliiita — APPROVED** [review]

File: styles/styles.css (hunk 1 above)
Comment (sdmcraft) [reply styles/styles.css:1168]: yes, that's what I found atleast through finder.

Comment (sdmcraft) [reply styles/styles.css:1168]: Also similar observation with Sharepoint search. There's also a `hero-banner(static)` variant but likely that's not related to this change as this one is limiter to section level style.

**Pull request merged**